addons/web/static/tests/core/notifications/notifications_tests.js

```diff
@@ -31,11 +31,12 @@ QUnit.test("can display a basic notification", async (assert) => {
     await nextTick();
     assert.containsOnce(target, ".o_notification");
     const notif = target.querySelector(".o_notification");
+    const notificationHeader = notif.querySelector(".toast-header");
```

Comment:
I remember when we discussed to scope the constant, but now that I see the entire code the creation of a new constants is actually not needed at all ^^
You can use the same approach as `product_pricelist_report_test.js`, so 

```js
assert.hasClass(notif.querySelector(".toast-header"), "border-warning");
```